Yosemite/YosemiteTests/Stores/Order/OrdersUpsertUseCaseTests.swift

```diff
@@ -227,4 +257,16 @@ private extension OrdersUpsertUseCaseTests {
               totalTax: "0.00",
               attributes: attributes)
     }
+
+    func makeOrderTaxLine() -> Networking.OrderTaxLine {
+        .init(taxID: 0,
+              rateCode: "TAX",
+              rateID: 0,
+              label: "Tax label",
+              isCompoundTaxRate: true,
+              totalTax: "0.0",
+              totalShippingTax: "0.0",
+              ratePercent: 0.0,
+              attributes: [])
```

Comment:
Similar to the comment above - this can be replaced with `.fake()` for simplicity.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/5931/commits/e0692c441b13f6a2d1ea7c5e7ed5b95db1c48d16